### ISSUE TITLE
Filter by type

### ISSUE THREAD

**Issue by BjornMelgaard** [ramda/ramda.github.io#188]
e.g. click on function type `Logic` -> site showing only functions of `Logic` type

**Comment by CrossEye** [comment]
Yes, that's how it works.  It fills the search field when you choose a category, and filters by that field.

Do you not like this?  I'm not quite sure what you're saying.

**Comment by BjornMelgaard** [comment]
Oh, I am sorry, now I see that this functionality already present in sidebar labels, but I was clicking on labels in main section. Maybe make them clickable too?

**Comment by customcommander** [comment]
I think what the issue is about is the labels on the main page is not clickable:

![Uploading Screenshot 2022-01-18 at 22.14.30.png…]()

**Comment by CrossEye** [comment]
I'm sure you're right.  And it makes sense.  It should be a simple enough fix.

At some point, it would be nice to write a more robust search, perhaps StackOverflow style:

```
name:andThen
```

```
[Function] then
```

```
onSuccess
```

Where clicking the "Function" category would add/remove `[Function]` from the search box, and where search terms are highlighted in the document, or even in a gloss on the matches.

This is not high priority, but it seems worth doing.
